### ISSUE TITLE
Ubuntu 17.10 Toolchain teminated with fatal error

### ISSUE THREAD

**Issue by XinzeZhang** [James-Yu/LaTeX-Workshop#478]
My operation system is Ubuntu 17.10, and latex workshop toolchain just work well several days ago.
May be after the recent update, I meet the "toolchain terminated with fatal error" when i build my tex project.
I can use xelatex\latexmk and so on the bash and terminal in the vs code. But the toolchain cannot wok normally anymore as it was fine before. It`s queer. Have anybody met this? Seek for help.

Cause the  "toolchain terminated with fatal error", i got nothing especially log file even aux file.  Hope for help, Thanks!

**Comment by James-Yu** [comment]
Interestingly enough, there is no recent update to the extension. The extension log (not compiler log) must said something.

**Comment by XinzeZhang** [comment]
I can using xelatex *.tex or latexmk *.tex, even the sequence of " xelatex *.tex; bibtex *.aux; xelatex *.tex; xelatex *.tex" in the bash or terminal in vs code. Where can i cat or find the extension log?

**Comment by James-Yu** [comment]
in vscode quick command

**Comment by XinzeZhang** [comment]
thanks, and any details? i cd in  the  /home/xinze/.vscode/extensions/James-Yu.latex-workshop-3.14.0"

**Comment by James-Yu** [comment]
You have the information when you create this issue.

```
<summary>LaTeX Workshop Output</summary>
<!-- To access the log, click the 'problems' icons on the statusbar, select 'OUTPUT', then select 'LaTeX Workshop' from the dropdown menu on the right. -->
```

**Comment by XinzeZhang** [comment]
Sorry, Thanks for help, I need to get out of the lab now, cause it's 11:20 pm now, our college building will close soon. I will commit log file tomorrow.  Thanks again.

**Comment by XinzeZhang** [comment]
The log of latex-workshop is committed as follow:

[12:29:19] Initializing LaTeX Workshop.
[12:29:19] Creating LaTeX Workshop http and websocket server.
[12:29:19] LaTeX Workshop initialized.
[12:29:19] Found root file from active editor: /home/xinze/Desktop/Research_Planning/research_planning.tex
[12:29:19] Root file changed from: undefined. Find all dependencies.
[12:29:19] Instatiating new file watcher for /home/xinze/Desktop/Research_Planning/research_planning.tex
[12:29:19] Parsing /home/xinze/Desktop/Research_Planning/research_planning.tex
[12:29:19] Adding /home/xinze/Desktop/Research_Planning/chapter1/section1.tex to file watcher.
[12:29:19] Parsing /home/xinze/Desktop/Research_Planning/chapter1/section1.tex
[12:29:19] Adding /home/xinze/Desktop/Research_Planning/chapter1/section2.tex to file watcher.
[12:29:19] Parsing /home/xinze/Desktop/Research_Planning/chapter1/section2.tex
[12:29:19] Adding /home/xinze/Desktop/Research_Planning/chapter1/section3.tex to file watcher.
[12:29:19] Parsing /home/xinze/Desktop/Research_Planning/chapter1/section3.tex
[12:29:19] Adding /home/xinze/Desktop/Research_Planning/chapter1/section4.tex to file watcher.
[12:29:19] Parsing /home/xinze/Desktop/Research_Planning/chapter1/section4.tex
[12:29:19] Adding /home/xinze/Desktop/Research_Planning/chapter1/section5.tex to file watcher.
[12:29:19] Parsing /home/xinze/Desktop/Research_Planning/chapter1/section5.tex
[12:29:19] Adding /home/xinze/Desktop/Research_Planning/chapter2/section1.tex to file watcher.
[12:29:19] Parsing /home/xinze/Desktop/Research_Planning/chapter2/section1.tex
[12:29:19] Adding /home/xinze/Desktop/Research_Planning/chapter2/section2.tex to file watcher.
[12:29:19] Parsing /home/xinze/Desktop/Research_Planning/chapter2/section2.tex
[12:29:19] Adding /home/xinze/Desktop/Research_Planning/chapter2/section3.tex to file watcher.
[12:29:19] Parsing /home/xinze/Desktop/Research_Planning/chapter2/section3.tex
[12:29:19] Adding /home/xinze/Desktop/Research_Planning/chapter2/section4.tex to file watcher.
[12:29:19] Parsing /home/xinze/Desktop/Research_Planning/chapter2/section4.tex
[12:29:19] Found .bib file /home/xinze/Desktop/Research_Planning/ref-research.bib
[12:29:19] Creating file watcher for .bib files.
[12:29:19] Parsing .bib entries from /home/xinze/Desktop/Research_Planning/ref-research.bib
[12:29:19] Parsed 15 .bib entries from /home/xinze/Desktop/Research_Planning/ref-research.bib.
[12:29:19] Linter for root file started.
[12:29:19] Linter for root file running command chktex with arguments -wall,-n22,-n30,-e16,-q,-f%f:%l:%c:%d:%k:%n:%m
,/home/xinze/Desktop/Research_Planning/research_planning.tex
[12:29:19] Linter for root file failed to spawn command, encountering error: spawn chktex ENOENT
[12:29:19] Server created on 127.0.0.1:46085
[12:29:19] LaTeX Workshop version: 3.14.0
[12:29:19] Default environments loaded
[12:29:19] Default commands loaded
[12:29:19] Default unimathsymbols loaded
[12:29:22] Parsing /home/xinze/Desktop/Research_Planning/research_planning.tex for outline
[12:29:22] Parsing /home/xinze/Desktop/Research_Planning/chapter2/section1.tex for outline
[12:29:22] Parsing /home/xinze/Desktop/Research_Planning/chapter2/section2.tex for outline
[12:29:22] Parsing /home/xinze/Desktop/Research_Planning/chapter2/section3.tex for outline
[12:29:22] Parsing /home/xinze/Desktop/Research_Planning/chapter2/section4.tex for outline
[12:29:22] BUILD command invoked.
[12:29:22] Found root file from active editor: /home/xinze/Desktop/Research_Planning/research_planning.tex
[12:29:22] Root file remains unchanged from: /home/xinze/Desktop/Research_Planning/research_planning.tex.
[12:29:22] Building root file: /home/xinze/Desktop/Research_Planning/research_planning.tex
[12:29:22] Build root file /home/xinze/Desktop/Research_Planning/research_planning.tex
[12:29:22] Toolchain step 1: xelatex, -synctex=1,-interaction=nonstopmode,-file-line-error,/home/xinze/Desktop/Research_Planning/research_planning
[12:29:22] LaTeX fatal error: spawn xelatex ENOENT, . Does the executable exist?
[12:33:13] Root file remains unchanged from: /home/xinze/Desktop/Research_Planning/research_planning.tex.

**Comment by XinzeZhang** [comment]
Thanks! I have solved this problem.

Cause there are two environment file in ubuntu, one is ~/.bashrc and the other is ~/.bash_profile. Several days ago, I created the ~/.bash_profile when i installed the sql server. The env-path of latex has been in the ~/.bashrc but not the ~/.bash_profile. 

So the extension cannot find the latex environment but i can get the right return when i execute the "xelatex --version" in bash or the terminal in vs code.

The solution is copy the env-path of latex from ~/.bashrc to ~/.bash_profile and reboot. Or delete the ~/.bash_profile and reboot.

Thanks!
